.infra/Pulumi.prod.yaml

```diff
@@ -38,7 +38,7 @@ config:
     defaultImageRatio:
       secure: AAABAHpff5rGIhX2EmH5QBZx1br4VVzK+JHKbbAUjYRT
     defaultImageUrl:
```

Comment:
Should I also change these values in adhoc file?

Reply:
Yeah, we want new pretty images in adhoc too

Reply:
Done